csrc/preseg_passes/insert_reshardings.cpp

```diff
@@ -110,7 +125,10 @@ void insertReshardingSetsAfter(Fusion* fusion) {
       ir_utils::replaceValInAllExprInputsAndFusionOutputs(output, new_output);
       // Update shardings new_output takes output's sharding,
       // output takes input's sharding
-      shardAllLike(output, {new_output}, deviceAndStreamParallelTypes());
+      std::unordered_set<ParallelType> parallel_types{
+          kParallelTypeDIDs.begin(), kParallelTypeDIDs.end()};
+      parallel_types.insert(ParallelType::Stream);
+      shardAllLike(output, {new_output}, parallel_types);
```

Comment:
https://github.com/NVIDIA/Fuser/blob/5ba57eebd9a3c609bc6f18742f60fd8019512494/csrc/multidevice/utils.h#L69 and remove getParallelTypesForResharding